File: abc/abc173/b.py
# https://atcoder.jp/contests/abc173/submissions/37555610

n = int(input())
ans = [0, 0, 0, 0]
ptn = ["AC", "WA", "TLE", "RE"]

while n:
  n -= 1
  s = input()
  if s == "AC":
    ans[0] += 1
  elif s == "WA":
    ans[1] += 1
  elif s == "RE":
    ans[3] += 1
  else:
    ans[2] += 1

for i in range(4):
  print(ptn[i], 'x', ans[i])

File: abc/abc173/b.java
// https://atcoder.jp/contests/abc173/submissions/43760832

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int[] cnt = new int[4];
    for(int i = 0; i < 4; i++) cnt[i] = 0;

    String[] ptn = {"AC", "WA", "TLE", "RE"};
    for(int i = 0; i < n; i++) {
      String s = stdIn.next();
      
      for(int j = 0; j < 4; j++) {
        if(s.equals(ptn[j])) {
          cnt[j]++;
          break;
        }
      }
    }

    System.out.printf("AC x %d\n", cnt[0]);
    System.out.printf("WA x %d\n", cnt[1]);
    System.out.printf("TLE x %d\n", cnt[2]);
    System.out.printf("RE x %d\n", cnt[3]);
  }
}

